packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -110,7 +115,7 @@ const fn = new lambda.Function(this, 'MyFunction', {
     runtime: lambda.Runtime.NODEJS_8_10,
     handler: 'index.handler',
     code: lambda.Code.fromInline('exports.handler = function(event, ctx, cb) { return cb(null, "hi"); }'),
-    deadLetterQueueEnabled: true
+    dlq: lambda.DeadLetterQueue.fromSqsQueue(),
```

Comment:
I think that in this case the `from` prefix doesn't feel right because we allow no parameters and so "from what?". I'd argue that `lambda.DeadLetterQueue.sqsQueue()` reads better in this case and cal also accomnodate `DeadLetterQueue.sqsQueue(x)` (@rix0rrr, thoughts?).

Alternatively we can do: `fromNewQueue()` and `fromQueue(x)`.